lib/manager/kustomize/readme.md

```diff
@@ -1,20 +1,22 @@
-This package will manage two parts of the `kustomization.yaml` file:
+This package will manage the following parts of the `kustomization.yaml` file:
 
 1. [remote bases](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md)
```

Comment:
```suggestion
1. [remote bases](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md) (deprecated since kustomize v2.1.0)
```

Should we add this note?

Reply:
I'm fine either way, your choice.

Reply:
Actually, bases are deprecated, resources are supposed to replace bases. Maybe we can change that to something like

```suggestion
1. [remote resources](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md)
```

Reply:
ok, move deprecated bases to last and add resources as first